src/main/java/fi/helsinki/cs/tmc/core/configuration/TmcSettings.java

```diff
@@ -3,6 +3,7 @@
 import fi.helsinki.cs.tmc.core.domain.Course;
 
 import com.google.common.base.Optional;
+import java.nio.file.Path;
```

Comment:
Yes, every group (saa @jamox above) should be separated by an empty line.

See [this](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing), but replace `com.google` with `fi.helsinki.cs.tmc`.